go.sum

```diff
@@ -54,20 +54,13 @@ github.com/Shopify/toxiproxy v2.1.4+incompatible h1:TKdv8HiTLgE5wdJuEML90aBgNWso
 github.com/Shopify/toxiproxy v2.1.4+incompatible/go.mod h1:OXgGpZ6Cli1/URJOF1DMxUHB2q5Ap20/P/eIdh4G0pI=
 github.com/acobaugh/osrelease v0.0.0-20181218015638-a93a0a55a249 h1:fMi9ZZ/it4orHj3xWrM6cLkVFcCbkXQALFUiNtHtCPs=
 github.com/acobaugh/osrelease v0.0.0-20181218015638-a93a0a55a249/go.mod h1:iU1PxQMQwoHZZWmMKrMkrNlY+3+p9vxIjpZOVyxWa0g=
-github.com/armakuni/dp-go-featuretest v0.0.0-20210211115457-6586e232612a h1:XdGBzEwI/Xlv8/hjrj39fibegM2F1rPSupVkNwj5qzY=
-github.com/armakuni/dp-go-featuretest v0.0.0-20210211115457-6586e232612a/go.mod h1:2zcapQgxd9Fuuf66ROP8aAWUBcMcrBpOezrs+jlpEkA=
-github.com/armakuni/dp-go-featuretest v0.0.0-20210211154136-f4f263cb8559 h1:Ksv2/TqVQabwkh4E3sZl7nA1S/CGhUM/Tri9Vq6Z9vE=
-github.com/armakuni/dp-go-featuretest v0.0.0-20210211154136-f4f263cb8559/go.mod h1:8PwfFu2YQNB3/if9L2VrrXyfEYf2NKOB6JInDYJKM/s=
+github.com/armakuni/dp-go-featuretest v0.0.0-20210225121618-47aebd5e92b6 h1:vbXjUtv6nagDev06BNtx5RWLRDPU1bUle3QQfJ2en08=
```

Comment:
Could we pull this into ONS' Github space?

Reply:
yes, the plan is to move it across soon - we initially made it under AK so we could start work prior to SC